src/main/java/org/llorllale/cactoos/matchers/MatchesBefore.java

```diff
@@ -49,22 +53,60 @@
  * @param <T> Type of the scalar's value
  * @since 1.0.0
  */
+@SuppressWarnings({
+    "PMD.AvoidCatchingGenericException",
+    "PMD.CallSuperInConstructor",
+    "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"
```

Comment:
Why these two are fired here?

Reply:
@andreoss who are you asking? It can be some old leftovers and be not needed, or maybe some qulice/pmd bug…

Reply:
@victornoel Seems to be a bug with Qulice